string[] names
float64[] confidences
float64[] x_positions
float64[] y_positions
---
string closest_plate
float64 confidence
float64 distance
